### PR TITLE
[sp] edit pipeline name functionality

### PR DESCRIPTION
**Pull request by shrey-mage**
# Summary
- see above

# Tests

cc: @johnson-mage @tommydangerous @wangxiaoyou1993 @dy46

### CODE REVIEW

File: mage_ai/frontend/pages/pipelines/[...slug].tsx [hunk 1]
```diff
@@ -317,6 +317,36 @@ function PipelineDetailPage({
...
+            console.log(errors, message);
+          },
+        },
+      );
+    },
+  )), [
+    fetchFileTree,
+    updatePipelineNoContent,
+    router,
+  ]);
```
Comment (dy46) [inline mage_ai/frontend/pages/pipelines/[...slug].tsx:348]: is there a reason you did this in 2 separate functions vs just using one `useMutation`?

**Review by dy46 — APPROVED** [review]

**Comment by johnson-mage** [comment]
@shrey-mage change the font color of the saved pipeline name back to white

**Comment by johnson-mage** [comment]
> @shrey-mage change the font color of the saved pipeline name back to white

looks like this rn:
image

File: mage_ai/frontend/pages/pipelines/[...slug].tsx (hunk 1 above)
Comment (shrey-mage) [reply mage_ai/frontend/pages/pipelines/[...slug].tsx:348]: I wasn't too familiar with this design pattern so I tried to emulate how it was done previously and did it separately to be safe. I can combine them

Comment (tommydangerous) [reply mage_ai/frontend/pages/pipelines/[...slug].tsx:348]: Did you merge this before fixing?

**Pull request merged**